ocaml/database/block_device_io.ml

```diff
@@ -383,13 +383,13 @@ let send_failure client prefix error =
 
 (* Read a byte from the given file descriptor. If it is not '|' then raises ExpectedSeparator. *)
 let read_separator client =
-  let str = "\000" in
+  let str = Bytes.of_string "\000" in
   Unixext.really_read client str 0 1;
-  if str <> "|" then raise ExpectedSeparator
+  if str <> (Bytes.of_string "|") then raise ExpectedSeparator
```

Comment:
Could this be written as `Bytes.get str 0 <> '|'?`

Reply:
Yes, I like this. Thanks!